Dockerfile

```diff
@@ -26,4 +26,4 @@ WORKDIR /oot
 RUN git config --global --add safe.directory /oot
 
 CMD ["/bin/sh", "-c", \
-    "echo 'usage:\n  docker run --rm --mount type=bind,source=\"$(pwd)\",destination=/oot oot make -j$(nproc) setup\n  docker run --rm --mount type=bind,source=\"$(pwd)\",destination=/oot oot make -j$(nproc)'"]
+    "make setup && make"]
```

Comment:
should we keep the `-j$(nproc)` to try to speed up the build here?

Reply:
I think we caught this at exact same time :D

Reply:
(never mind)